content/en/docs/collector/benchmarks.md

```diff
@@ -3,75 +3,74 @@ title: Benchmarks
 weight: 99
 ---
```

Comment:
Best to review this file while ignoring whitespace in the diff. While the HTML was unindented, only a few other tweaks were done to this page.